### PR TITLE
Merging first version of the Newsweek clone

### PR DESCRIPTION
**Pull request by o-rashad**
We did use :  

- SASS for styling.
- Bootstrap 4 for positioning and layouts three screen sizes ( small, medium, and large )
- Linters for code testing 
- Dev tools for inspecting

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1208 @@
...
+                <li class="list-inline-item"><a class="text-black-50 font-weight-bold px-3" href="#">CORONAVIRUS</a></li>
+                <li class="list-inline-item"><a class="text-black-50 font-weight-bold px-3" href="#">COUNTIES THAT COUNT</a></li>
+            </ul>
+        </div>
+
+    </nav>
+
+    <!-- MAIN CONTAINER-->
+    <div class="container-fluid main-content">
+        <div class="top-story d-sm-block d-md-none">
```
Comment (uche-inyama) [inline index.html]: The main section should be wrapped between the **main tag** [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1208 @@
...
+            <li class="list-inline-item"><a href="#">Cookie Policy</a></li>
+            <li class="list-inline-item"><a href="#">Terms of Sale</a></li>
+            <li class="list-inline-item"><a href="#">Archive</a></li>
+            <li class="list-inline-item"><a href="#">Announcements</a></li>
+        </ul>
+
+    </div>
+
+    <!-- <div class="cir">
+    </div> -->
```
Comment (uche-inyama) [inline index.html]: Remove commented lines of code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1208 @@
...
+
+            </div>
+
+        </div>
+
+    </div>
+
+    <!-- FOOTER -->
+    <div class="container footer">
+
```
Comment (uche-inyama) [inline index.html]: The footer section should be wrapped between a **footer tag**. [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required 

### Hi Omar :wave: 

Great work so far, I must commend your efforts :+1: 

But there are things we have to fix: 

 **@media size >= 1024px** 
- [x]  add thumbnails to close up the spaces after **Opinion** and **Featured Stories** sections, so they look like real page.

- [x] The images on the right side of the section after, **Editor's Pick** are stretching. I would suggest you crop it to fit the size required, before applying it to the page.

 **@media 768px**
- [x] add padding to the right side of each article in the **More Stories** section.
- [x] add thumbnails after the **Featured Stories** to cover the empty space.
-  [x] Between the **Navigation bar** to **Culture & Travel** section, there is a space at the right side of the page that needs to be re-adjusted, so it looks like the real page.
- [x] The images on the right side of the section after, **Editor's Pick** are stretching. I would suggest you crop the images to fit the size required before applying it.

**@media >=403px & < 768px**

- [x] The thumbnails between **Top Stories** & **My Turn** should be centered properly.


- [x] Use [markdown](https://guides.github.com/features/mastering-markdown/) to format the texts in your README file

#### That's it, when you are done with these changes submit another form for a code review

#### If you need any clarifications, please do reach out on slack.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Comment by o-rashad** [comment]
Thanks, @uche-inyama for the well-crafted review ⭐️⭐️⭐️
We (me and @apricot12 ) are currently working on it.
considering submitting the changes ASAP.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes Requested 🔴 

### Hello @od-c0d3r and @apricot12 👋 

### Thank you for making almost all of the requested changes from the previous TSE, but I believe you missed one of them:

- [x] On big and medium screens, in the "Tech", "Sports", "USA", etc section, we can see that the images to the right are still stretched. Make sure you crop those images, or even better, work on their width and height in your CSS, so they are proportional even when the width of the page changes. Take a look below:

Sem título


Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by o-rashad** [comment]
Thanks, @MuriloRoque for the review ⭐️⭐️⭐️
We are currently working on it.

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi team  :wave: 

Move on with your next task. 

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Pull request merged**